packages/panzer/disc-fe/src/Panzer_Workset.cpp

```diff
@@ -208,7 +255,20 @@ WorksetDetails::getIntegrationRule(const panzer::IntegrationDescriptor & descrip
 }
 
 const panzer::BasisValues2<double> &
-WorksetDetails::getBasisValues(const panzer::BasisDescriptor & basis_description, const panzer::IntegrationDescriptor & integration_description) const
+WorksetDetails::getBasisValues(const panzer::BasisDescriptor & basis_description, 
+                               const panzer::PointDescriptor & point_description) const
+{
+  const auto itr = _basis_map.find(basis_description.getKey());
+  TEUCHOS_ASSERT(itr != _basis_map.end());
```

Comment:
Lets swap these TEUCHOS_ASSERT calls with one of the TEUCHOS_TEST_FOR_EXCEPT message calls so we know what basis/integration/point descriptor is missing. We can do this later though.